### PR TITLE
Bug 1499856 - new cert without pulse.taskcluster.net

### PR DESCRIPTION
**Pull request by djmitche**
I've already flagged brian for like 732 PRs in the last week, so I'm sharing the love.

### CODE REVIEW

**Review by ajvb — APPROVED** [review]

**Comment by ajvb** [comment]
@djmitche Can't make it green without write access, but lgtm

**Comment by djmitche** [comment]
Huh, I keep learning about new github features :)

**Comment by djmitche** [comment]
I applied this about an hour ago.

**Pull request merged**
